tests/bins/te/Readme.md

```diff
@@ -0,0 +1,18 @@
+# TE binaries

```

Comment:
thanks for adding this readme! Please rename the file to README.md to match other readmes though :)

Reply:
Updated README.md to match the standard of the repository. Thanks!